docs/troubleshoot/app-framework/app-troubleshoot.md

```diff
@@ -1,8 +1,194 @@
 # Troubleshooting Zowe Application Framework
 
-The following topics contain information that can help you troubleshoot problems when you encounter unexpected behavior installing and using Zowe&trade; Application Framework.
+The following topics contain information that can help you troubleshoot problems when you encounter unexpected behavior installing and using Zowe&trade; Application Framework which includes the Zowe Desktop.
 
-Before you reach out for support:
+Most of the solutions below identify issues by referring to the Zowe [logs](app-mustgather.md). To identify and resolve issues, you should become familiar with their names and locations.
 
-1. Is there already a GitHub issue (open or closed) that covers the problem? Check [Application Framework issues](https://github.com/zowe/zlux/issues).
-2. Review the current list of [Known issues](app-known-issues.md) in documentation. Also try searching using the Zowe Docs search bar.
+The Zowe Application Framework uses GitHub issues.  Check whether is an already a GitHub issue (open or closed) that covers the problem, see [https://github.com/zowe/zlux/issues](https://github.com/zowe/zlux/issues).

```

Comment:
"Check whether is an already" should be "Check whether there already is", or better still "Check whether a GitHub issue (open or closed) that covers the problem already exists".